jsons/Policies.json

```diff
@@ -985,7 +995,7 @@
             {
                 "name": "Creative Expression",
                 "uniques": [
-                    "[+2 Culture] from every [Great Work]",
+                    "[+3 Culture] from every [Great Work]",
```

Comment:
I am not certain where the "3" comes from, unless I have this policy just wrong. I would need to double check

Reply:
Ok, after double checking, this does look to only give 2 culture. Where is the third coming from?

Reply:
There +2 culture, +2 turism, tried to replace turism..